SC-XX: Sunset the Inclusion of Address and Routing Parameter Area Names (#573)
cabforum/servercert: docs/BR.md

| 2025-03-15     | 3.2.2.9                   | CAs MUST corroborate the results of domain validation and CAA checks from multiple Network Perspectives where specified.                                                                                                                                                                                                                                                                                                                                 |
| 2025-07-15     | 3.2.2.4                   | CAs MUST NOT rely on Methods 3.2.2.4.2 and 3.2.2.4.15 to issue Subscriber Certificates.                                                                                                                                                                                                                                                                                                                                                                  |

> | 2025-09-15     | 4.2.2                     | CAs SHALL NOT issue Certificates containing Address and Routing Parameter Area Names.                                                                                                                                                                                                                                                                                                                                                                         |

Reviewer: ```suggestion
| 2026-03-15     | 4.2.2                     | CAs SHALL NOT issue Certificates containing Address and Routing Parameter Area Names.                                                                                                                                                                                                                                                                                                                                                                         |
```

Suggest future effective date

| 2026-03-15     | 3.2.2.8.1                 | CAs MUST NOT use local policy to disable DNSSEC validation on any DNS query associated CAA record lookups.                                                                                                                                                                                                                                                                                                                                               |
| 2026-03-15     | 3.2.2.8.1                 | DNSSEC-validation errors observed by the Primary Network Perspective (e.g., SERVFAIL) MUST NOT be treated as permission to issue.                                                                                                                                                                                                                                                                                                                        |

> | 2026-03-15     | 4.2.2                     | CAs SHALL NOT issue Certificates containing IP Reverse Address Domain Names.                                                                                                                                                                                                                                                                                                                                                                         |

Reviewer 2: ```suggestion
| 2026-03-15     | 4.2.2                     | CAs SHALL NOT issue Certificates containing Domain Names ending in an IP Reverse Zone Suffix.                                                                                                                                                                                                                                                                                                                                                                         |
```

### 1.6.1 Definitions

> **Address and Routing Parameter Area Name**: A Domain Name whose Top-Level Domain is "arpa". Example: `1.2.0.192.in-addr.arpa`.

Reviewer: Shall we also add an example for ip6.arpa?

Reviewer: 6.4.0.0.1.0.0.0.7.0.2.0.5.5.1.0.1.0.0.0.0.0.8.2.8.4.6.0.1.0.0.2.ip6.arpa. :-)

Author: Added :)

Reviewer 3: It would be better to use a prefix for documentation like as IPv4 (`192.0.2.1`). The prefix is `2001:db8::/32` ([RFC 3849](https://datatracker.ietf.org/doc/html/rfc3849)).
Example: `1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.d.e.e.f.8.b.d.0.1.0.0.2.ip6.arpa.`

> **Address and Routing Parameter Area Name**: A Domain Name whose Top-Level Domain is "arpa". Examples: `1.2.0.192.in-addr.arpa` (IP version 4) and `1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.8.b.d.0.1.0.0.2.ip6.arpa` (IP version 6).

Reviewer: ```suggestion
**Address and Routing Parameter Area Name**: A Domain Name whose Top-Level Domain ends with the labels "in-addr.arpa" or "ip6.arpa". Examples: `1.2.0.192.in-addr.arpa` (IP version 4) and `1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.8.b.d.0.1.0.0.2.ip6.arpa` (IP version 6).
```

Reviewer 4: ```suggestion
**Address and Routing Parameter Area Name**: A Domain Name which ends with the labels "in-addr.arpa" or "ip6.arpa". Examples: `1.2.0.192.in-addr.arpa` (IP version 4) and `1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.8.b.d.0.1.0.0.2.ip6.arpa` (IP version 6).
```

**Application Software Supplier**: A supplier of Internet browser software or other relying-party application software that displays or uses Certificates and incorporates Root Certificates.

> **Address and Routing Parameter Area Name**: A Domain Name whose Top-Level Domain is "arpa". Example: `1.1.168.192.in-addr.arpa`.

Reviewer: The language is clear but perhaps the example makes the reader believe we are trying to block only non-routable IP addresses. If this is not a big concern by others, you may ignore this comment :)

Author: That's a good point. Maybe we can switch to an IP address from one of these blocks: https://datatracker.ietf.org/doc/html/rfc5737#section-3

### 1.6.1 Definitions

> **Address and Routing Parameter Area Name**: A Domain Name whose Top-Level Domain ends with the labels "in-addr.arpa" or "ip6.arpa". Examples: `1.2.0.192.in-addr.arpa` (IP version 4) and `1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.8.b.d.0.1.0.0.2.ip6.arpa` (IP version 6).

Reviewer 4: While the definition correctly captures the scope for reverse DNS names, the name "Address and Routing Parameter Area Name" sounds like it's intended to apply to all of `.arpa`. There's a risk of confusion.

Suggest renaming this definition to "IP reverse address domains".

(The change would affect other places in this PR as well.)

### 4.2.2 Approval or rejection of certificate applications

CAs SHALL NOT issue Certificates containing Internal Names or Reserved IP Addresses, as such names cannot be validated according to [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) or [Section 3.2.2.5](#3225-authentication-for-an-ip-address).

> Effective 2025-09-15, CAs SHALL NOT issue Certificates containing Address and Routing Parameter Area Names.

Reviewer: ```suggestion
Effective 2026-03-15, CAs SHALL NOT issue Certificates containing Address and Routing Parameter Area Names.
```

| ---                         | -             | ------         |
| `otherName`                 | N             | -              |
| `rfc822Name`                | N             | -              |

> | `dNSName`                   | Y             | The entry MUST contain either a Fully-Qualified Domain Name or Wildcard Domain Name that the CA has validated in accordance with [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control). Wildcard Domain Names MUST be validated for consistency with [Section 3.2.2.6](#3226-wildcard-domain-validation). The entry MUST NOT contain an Internal Name. Effective 2025-09-15, the entry MUST NOT contain an Address and Routing Parameter Area Name. The Fully-Qualified Domain Name or the FQDN portion of the Wildcard Domain Name contained in the entry MUST be composed entirely of P-Labels or Non-Reserved LDH Labels joined together by a U+002E FULL STOP (".") character. The zero-length Domain Label representing the root zone of the Internet Domain Name System MUST NOT be included (e.g. "example.com" MUST be encoded as "example.com" and MUST NOT be encoded as "example.com."). |

Reviewer: ```suggestion
| `dNSName`                   | Y             | The entry MUST contain either a Fully-Qualified Domain Name or Wildcard Domain Name that the CA has validated in accordance with [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control). Wildcard Domain Names MUST be validated for consistency with [Section 3.2.2.6](#3226-wildcard-domain-validation). The entry MUST NOT contain an Internal Name. Effective 2026-03-15, the entry MUST NOT contain an Address and Routing Parameter Area Name. The Fully-Qualified Domain Name or the FQDN portion of the Wildcard Domain Name contained in the entry MUST be composed entirely of P-Labels or Non-Reserved LDH Labels joined together by a U+002E FULL STOP (".") character. The zero-length Domain Label representing the root zone of the Internet Domain Name System MUST NOT be included (e.g. "example.com" MUST be encoded as "example.com" and MUST NOT be encoded as "example.com."). |
```

| ---                         | --            | -----          |
| `otherName`                 | N             | -              |
| `rfc822Name`                | N             | -              |

> | `dNSName`                   | Y             | The entry MUST contain either a Fully-Qualified Domain Name or Wildcard Domain Name that the CA has validated in accordance with [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control). Wildcard Domain Names MUST be validated for consistency with [Section 3.2.2.6](#3226-wildcard-domain-validation). The entry MUST NOT contain an Internal Name. Effective 2026-03-15, the entry MUST NOT contain an IP Reverse Address Domain Name. The Fully-Qualified Domain Name or the FQDN portion of the Wildcard Domain Name contained in the entry MUST be composed entirely of P-Labels or Non-Reserved LDH Labels joined together by a U+002E FULL STOP (".") character. The zero-length Domain Label representing the root zone of the Internet Domain Name System MUST NOT be included (e.g. "example.com" MUST be encoded as "example.com" and MUST NOT be encoded as "example.com."). |

Reviewer 2: ```suggestion
| `dNSName`                   | Y             | The entry MUST contain either a Fully-Qualified Domain Name or Wildcard Domain Name that the CA has validated in accordance with [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control). Wildcard Domain Names MUST be validated for consistency with [Section 3.2.2.6](#3226-wildcard-domain-validation). The entry MUST NOT contain an Internal Name. Effective 2026-03-15, the entry MUST NOT contain a Domain Name ending in an IP Reverse Zone Suffix. The Fully-Qualified Domain Name or the FQDN portion of the Wildcard Domain Name contained in the entry MUST be composed entirely of P-Labels or Non-Reserved LDH Labels joined together by a U+002E FULL STOP (".") character. The zero-length Domain Label representing the root zone of the Internet Domain Name System MUST NOT be included (e.g. "example.com" MUST be encoded as "example.com" and MUST NOT be encoded as "example.com."). |
```
